shared/subspace-data-retrieval/Cargo.toml

```diff
@@ -20,13 +20,13 @@ parity-scale-codec = { workspace = true, features = ["derive"] }
 subspace-archiving.workspace = true
 subspace-core-primitives = { workspace = true, features = ["std"] }
 subspace-erasure-coding.workspace = true
-subspace-runtime-primitives = { workspace = true, features = ["std"] }
+# This crate can't depend on any runtime code, because it needs to be independent of Substrate.
```

Comment:
seems surprising adding this comment here. Maybe add it a note in the module itself ?

Reply:
I've made this mistake multiple times in different Rust modules in this crate, so there isn't any module where a similar comment would prevent the issue. But any module that adds the dependency will have to do it right here.